modules/lambda@edge/variables.tf

```diff
@@ -29,6 +29,8 @@ variable "functions" {
     source_zip   = optional(string)
     runtime      = string
     handler      = string
+    memory_size  = optional(number)
```

Comment:
@mihaiplesa it could be handy to add a link to https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-at-edge-function-restrictions.html somewhere here to note that there are  specific Lambda@Edge limitations.

Reply:
Added.